tools/bcl-test-importer/BCLTestImporter/BCLTestProjectGenerator.cs

```diff
@@ -445,6 +445,44 @@ internal static IEnumerable<string> GetIgnoreFiles (string templatePath, string
 				}
 			}
 		}
+		
+		internal IEnumerable<string> GetTraitsFiles (Platform platform)
+		{
+			var rootPath = GetReleaseDownload (platform);
+			switch (platform) {
+			case Platform.iOS:
+				rootPath = Path.Combine (rootPath, "ios-bcl", "monotouch", "tests");
+				break;
+			case Platform.TvOS:
+				rootPath = Path.Combine (rootPath, "ios-bcl", "monotouch_tv", "tests");
+				break;
+			case Platform.WatchOS:
+				rootPath = Path.Combine (rootPath, "ios-bcl", "monotouch_watch", "tests");
+				break;
+			case Platform.MacOSFull:
+				rootPath = Path.Combine (rootPath, "mac-bcl", "xammac_net_4_5", "tests");
+				break;
+			case Platform.MacOSModern:
+				rootPath = Path.Combine (rootPath, "mac-bcl", "xammac", "tests");
+				break;
+			}
+			foreach (var fileName in new [] {"nunit-excludes.txt", "xunit-excludes.txt"}) {
```

Comment:
There's no need for an iterator, you can simply return an array:

```csharp
return new [] {
    Path.Combine (rootPath, "nunit-excludes.txt"),
    Path.Combine (rootPath, "xunit-excludes.txt")
};
```

which is faster and uses less memory.

Reply:
that was the python developer in me, I'll fix that :)